src/pipeline_v3/tests/integration/test_llama_index_availability.py

```
@@ -0,0 +1,74 @@
+"""Test to verify LlamaIndex availability in CI environment."""
+
+import pytest
+
+
+def test_llama_index_imports():
+    """Test that all required LlamaIndex imports work."""
+    errors = []
+
+    # Test each import individually
+    try:
+        import qdrant_client
```

Ruff (0.11.9)

`qdrant_client` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)